src/src.pro

```diff
@@ -163,6 +163,9 @@ packagesExist(contentaction5) {
     PKGCONFIG += contentaction5
     DEFINES += HAVE_CONTENTACTION
 } else {
+    PKGCONFIG += \
+        gio-2.0 \
+        gio-unix-2.0
```

Comment:
Better to add these dependencies to spec as well.

Reply:
This the fallback for builds without libcontentaction, which is a dependency defined in the spec so this branch won't be evaluated.

I don't know why it's an optional dependency.

Reply:
Ok, got it.

Reply:
You could add a build condition to build with either of them.
See:
http://rpm.org/user_doc/conditional_builds.html